docs/modules/ROOT/pages/tmail-backend/jmap-extensions/calendarEventParsing.adoc

```diff
@@ -0,0 +1,137 @@
+= Calendar event parse
+:navtitle: calendar event parse
+
+Allows users to parse calendar events for getting event information from the blob id of the calendar file (.ics).
+
+== Additions to the capability object
+
+Servers supporting the calendar event parse extension need
+to advertise it through the session capabilities:
+....
+com:linagora:params:calendar:event
+....
+
+The associated object is empty.
+
+== CalendarEvent/parse
+
+This method bases on:
+- Parsing a Received MDN method as described in the RFC-9007
+- JMAP CalendarEvent RFC-8620
```

Comment:
jmap calendar is still a draft (RFC8620 is the core spec), so wrong link here

-> https://datatracker.ietf.org/doc/html/draft-ietf-jmap-calendars